manifests/ProcessHacker/ProcessHacker2/2.39.0.0.yaml

```diff
@@ -0,0 +1,20 @@
+Id: ProcessHacker.ProcessHacker2
+Version: 2.39.0.0
+Name: ProcessHacker2
+Publisher: ProcessHacker
+License: GPLv3
+LicenseUrl: https://github.com/processhacker/processhacker/blob/master/LICENSE.txt
+AppMoniker: processhacker2
+Tags: taskview
+Description: A free, powerful, multi-purpose tool that helps you monitor system resources, debug software and detect malware.
+Homepage: https://processhacker.sourceforge.io/
+Installers:
+  - Arch: x86
+    InstallerType: EXE
+    Url: https://github.com/processhacker/processhacker/releases/download/v2.39/processhacker-2.39-setup.exe
+    Sha256: 28042DD4A92A0033B8F1D419B9E989C5B8E32D1D2D881F5C8251D58CE35B9063
+    Language: En
+    Switches:
+      Silent: /VERYSILENT
+      SilentWithProgress: /VERYSILENT
+# Generated by https://github.com/ptorr-msft/WinGetYamlGenerator
```

Comment:
```suggestion
Installers:
  - Arch: x86
    InstallerType: inno
    Url: https://github.com/processhacker/processhacker/releases/download/v2.39/processhacker-2.39-setup.exe
    Sha256: 28042DD4A92A0033B8F1D419B9E989C5B8E32D1D2D881F5C8251D58CE35B9063
# Generated by https://github.com/ptorr-msft/WinGetYamlGenerator
```

use the built in inno installer type so that the switches and language flags are not necessary.